src/main/java/com/example/diplomnabe/Controllers/UserController.java

```diff
@@ -1,14 +1,16 @@
 package com.example.diplomnabe.Controllers;
 
 import com.example.diplomnabe.Classes.User;
+import com.example.diplomnabe.DTO.UserDTO;
 import com.example.diplomnabe.Services.UserService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.ArrayList;
 import java.util.List;
 
 @RestController
-@RequestMapping(path = "api/version1/user")
+@RequestMapping(path = "api/version1/users")
```

Comment:
Do not write urls like this.
If you want to implement versioning into your route have a look at
https://stackoverflow.com/questions/20198275/how-to-manage-rest-api-versioning-with-spring

But for your graduation project I would exclude it from the routes